server/sms/sms_test.go

```diff
@@ -0,0 +1,24 @@
+package gateway
+
+import (
+	"github.com/smancke/guble/testutil"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNexmoSender_Send(t *testing.T) {
```

Comment:
This is an integration test? Will this send SMS directly to nexmo?
It should mock the nexmo api if possible.